docs/vehicle_model.md

```diff
@@ -0,0 +1,24 @@
+# Vehicle Dynamic model
```

Comment:
This has been merged on a separate PR, something weird is going on

Reply:
Sorry!! Fixed now.